src/Paramore.Brighter.ServiceActivator/ControlBus/ControlBusReceiverBuilder.cs

```diff
@@ -183,6 +185,7 @@ public Dispatcher Build(string hostName)
             commandProcessor = CommandProcessorBuilder.StartNew()
                 .Handlers(new HandlerConfiguration(subscriberRegistry, new ControlBusHandlerFactorySync(_dispatcher, () => commandProcessor)))
                 .Policies(policyRegistry)
```

Comment:
I wonder if we need a single configuration endpoint that we can pass both into, making them optional, and then determine which one to use. I think the separate APIs may be more confusing than an API that documents you can add old school, new school, or both